### DISCUSSION TITLE
A way to change default combo colors on the skin editor

### DISCUSSION THREAD

**Discussion by sininen-blue** [ppy/osu#23193] [Ideas]
This was done in stable through the skin config file, 

and it would be nice if we could change it without going through the process of importing it on stable, editing the file, and then importing it to lazer

Ideally this would also work on argon since I really like the skin, but find some of the colors hard to read. (though I'm not exactly sure if they work on the same system)

**Comment by ItsShamed** [comment]
You probably have to wait until osu!lazer gets its own skinning workflow, with a fully-fledged editor which would probably include this and other features such as metadata edition, hitsounds, gameplay cursor etc. (https://github.com/ppy/osu/discussions/18314#discussioncomment-5426217)
